src/core/public/plugins/plugin_loader.ts

```diff
@@ -20,20 +20,14 @@
 import { PluginName } from '../../server';
 import { PluginInitializer } from './plugin';
 
-/**
- * Unknown variant for internal use only for when plugins are not known.
- * @internal
- */
-export type UnknownPluginInitializer = PluginInitializer<unknown, Record<string, unknown>>;
-
 /**
  * Custom window type for loading bundles. Do not extend global Window to avoid leaking these types.
  * @internal
  */
 export interface CoreWindow {
   __kbnNonce__: string;
   __kbnBundles__: {
-    [pluginBundleName: string]: UnknownPluginInitializer | undefined;
+    [pluginBundleName: string]: object | undefined;
```

Comment:
Not sure it's a correct.`__kbnBundles__` contains init functions, not objects.
https://github.com/restrry/kibana/blob/af295b43bdf27bcefd7ed293fa97e96a9656add0/src/core/public/plugins/plugin_loader.ts#L91-L99
```suggestion
    [pluginBundleName: string]: PluginInitializer | undefined;
```

Reply:
That actually illustrates another problem with the current code. any not primitive is a valid type
![2019-07-04_10-14-30](https://user-images.githubusercontent.com/3198181/60650529-9bc73800-9e44-11e9-8c5c-e8f3552937ed.png)